.gitconfig

```diff
@@ -0,0 +1,17 @@
+[user]
+  name = Kanika Wadhawan
+  email = wadhawan.kanika@gmail.com
+[diff]
+[color.diff]
+  meta = yellow
+  new = green reverse
+  whitespace = red reverse
+[color]
+    ui = auto
+[alias]
+    st = status
+    ci = commit --verbose
+    co = checkout
+    di = diff -w
+    dc = diff --cached -w
+    noff = merge --no-ff
```

Comment:
This file should be in the $HOME dir. Please refer to https://github.com/redacademy/wr-fall-2019/wiki/Git-Merge-Workflow

Reply:
I think you are not able to use this aliases at the moment. Please check.

Reply:
Sure, thank you.